Services/UI/OJS.Services.Ui.Models/Contests/ContestLegacyExportServiceModel.cs

```diff
@@ -0,0 +1,213 @@
+namespace OJS.Services.Ui.Models.Contests;
+
+using AutoMapper;
+using Data.Models;
+using Data.Models.Checkers;
+using Data.Models.Contests;
+using Data.Models.Problems;
+using Data.Models.Submissions;
+using Data.Models.Tests;
+using Infrastructure.Models.Mapping;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class ContestLegacyExportServiceModel : IMapExplicitly
+{
+    public int Id { get; set; }
+
+    public string Name { get; set; }
+
+    public bool IsVisible { get; set; }
+
+    public DateTime? VisibleFrom { get; set; }
+
+    public bool AutoChangeTestsFeedbackVisibility { get; set; }
+
+    public DateTime? StartTime { get; set; }
+
+    public DateTime? EndTime { get; set; }
+
+    public string? NewIpPassword { get; set; }
+
+    public DateTime? PracticeStartTime { get; set; }
+
+    public DateTime? PracticeEndTime { get; set; }
+
+    public int CategoryId { get; set; }
+
+    public int Type { get; set; }
+
+    public TimeSpan? Duration { get; set; }
+
+    public string? ContestPassword { get; set; }
+
+    public string? PracticePassword { get; set; }
+
+    public int LimitBetweenSubmissions { get; set; }
+
+    public int OrderBy { get; set; }
+
+    public short NumberOfProblemGroups { get; set; }
+
+    public string? Description { get; set; }
+
+    public bool UsersCantSubmitConcurrently { get; set; }
+
+    public int DefaultWorkerType { get; set; }
+
+    public bool EnsureValidAuthorSubmisions { get; set; }
+
+    public List<LegacyProblemGroup> ProblemGroups { get; set; }
```

Comment:
Same here, we can initialize the collection like this: `public List<LegacyProblemGroup> ProblemGroups { get; set; } = [];`